napari/layers/image/image_update_contract.py

```diff
@@ -0,0 +1,22 @@
+from abc import abstractmethod
+from ..base._base_update_contract import UpdateContractBase
+
+
+update_methods = {"interpolation": "_set_interpolation"}
```

Comment:
Why not just have the convention that we always prepend '_set_' to the method name? 
That would reduce boiler code here and, impoortantly I think,  avoid the need to maintain 
such maps that are not analysed and checks for errors by IDEs....

Reply:
That  would let you push that code up to the abstract class...

Reply:
yea that'd work! I like it

Reply:
Updated with this change